antarest/study/storage/rawstudy/watcher.py

```diff
@@ -132,6 +133,10 @@ def _rec_scan(
                 logger.info(f"No scan directive file found. Will skip further scan of folder {path}")
                 return []
 
+            if is_temporary_upgrade_dir(path):
```

Comment:
You should add the same check for time series generation.
You can do the same thing you did in `antares-study-version` but for the TS gen inside this repo.